docs/_templates_system/templates.rst

```diff
@@ -6,7 +6,39 @@ Templates
 Gimme and SWP custom Twig tags
 ------------------------------
 
-Gimme allows you fetch nedded Meta object in any place of your template file. It supports single Meta objects (with :code:`gimme` ) and collections of Meta objects (with :code:`gimme_list`).
+Gimme allows you fetch nedded Meta object in any place of your template file. It supports single Meta objects (with :code:`gimme` ) and collections of Meta objects (with :code:`gimmelist`).
+
+container
+`````````
+
+Tag :code:`container` have one required and one optional parameters:
+
+ * (required) container unique name ex.: *frontpage_sidebar*
+ * (optional) keyword :code:`with` and default parameters for containers (they are used to create container on theme instalation).
+
+.. code-block:: twig
+
+     {% container 'frontpage_sidebar' with {
+         'width': 400,
+         'height': 500,
+         'styles': 'border: solid 1px red',
+         'class': 'css_class_name',
+         'data': {'custom-key': value}
+     }%}
+     {% endcontainer %}
+
+This container tag will render that html code:
+
+.. code-block:: html
+
+    <div id="frontpage_sidebar" class="swp_container css_class_name" style="width: 300px; height: 500px; border: solid 1px red;" data-custom-key="value"></div>
+
+Available container parameters:
+ * [integer] width - container width
+ * [integer] height - container height
+ * [string] styles - container inline styles
+ * [string] class - container class string
+ * [string] data - json object string with html-data properties (keys and values)
```

Comment:
Nope, just one level. But would be smart to put json string as data value if value is also an array/object